accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -680,18 +680,48 @@ It's a compile-time error if:
     class M with S {} // Error, for several reasons.
     ```
 
-*   A class extends or mixes in a declaration marked `interface` or `final`
+*   A declaration depends directly on a `final` declaration from another
+    library _(with some exceptions for platform libraries)_.
+
+    _(You cannot subtype a `final` declaration, except inside the same library.
+    Unless you are in a pre-feature library and the super-declaration is from
+    a platform library.)_
+
+    More formally, it's a compile-time error if:
+    A `class`, `mixin class`, `mixin` or `enum` declaration from library *L*
+    has a direct superdeclaration *S* marked `final` _(so necessarily a `class`
+    declaration)_ in a library *K*, neither:
```

Comment:
I would do "neither" -> "and neither of the following is true".

And then remove the "nor," from the first bullet.

Likewise further below.